src/zjs_modules.c

```diff
@@ -32,28 +33,27 @@ struct routine_map {
 static u8_t num_routines = 0;
 struct routine_map svc_routine_map[NUM_SERVICE_ROUTINES];
 
+    static jerryx_module_resolver_t resolvers[1] =
```

Comment:
If we go with the final version of the module API, this needs to change to
```c
static const jerryx_module_resolver_t *resolvers[1] = {
  &jerryx_module_native_resolver
};
```
because in the new version of the API the symbol `jerryx_module_native_resolver` refers to a global structure, not a function pointer.

Reply:
Thanks, updated